core/drivers/crypto/se050/glue/include/i2c_native.h

```diff
@@ -9,8 +9,25 @@
 
 #include <kernel/rpc_io_i2c.h>
 
+#if defined(CFG_CORE_SE05X_I2C_TRAMPOLINE) && CFG_CORE_SE05X_I2C_TRAMPOLINE
```

Comment:
`#ifdef CFG_CORE_SE05X_I2C_TRAMPOLINE` only

Reply:
Can we undef a config option? Since`core/drivers/crypto/se050/crypto.mk` sets `CFG_CORE_SE05X_I2C_TRAMPOLINE` to `y`, it will show up in the generated `conf.h`.

Reply:
Boolean `CFG_xxx` config switches seen by source files are expected to be defined when the switch is enabled (`y`), and not defined when disabled (`n`).

If your platform needs to have `CFG_CORE_SE05X_I2C_TRAMPOLINE` disabled, it can simply use
`CFG_CORE_SE05X_I2C_TRAMPOLINE ?= n` or `$(call force,CFG_CORE_SE05X_I2C_TRAMPOLINE,n)`
in its _plat-xxx/conf.mk_ file
Or alternatively set the config from your build command line:
`make PLATFORM=... CFG_CORE_SE05X_I2C_TRAMPOLINE=n`.

Reply:
done